### ISSUE TITLE
Problem with gen_name in Normaliz. Help needed!! 

### ISSUE THREAD

**Issue by w-bruns** [videlec/e-antic#124]
I am trying to make Normaliz (presently using e-natic 0.1.7) a little more flexible by allowing a name != a for the generator of the number field. For some reason I could not succeed. Most likely something is suboptimal on the Normaliz side. In input.cpp I have 
```
   renf = renf_class(mp_string, indet, emb_string);
    
    cout << "indet " << indet << " gen_name " << renf.gen_name << endl;
```
Result by running an input file
`indet b gen_name a`

What really puzzles me is that at the end of the renf constructor used things seem to be o.k.:

`var b gen_name b`

as a result of

```
    ...
   fmpq_poly_clear(p);
    arb_clear(e);
    
    std::cout << "var " << var << " gen_name " << this->gen_name << std::endl;
}
```
The update to 0.1.8 has not helped. Where is my mistake?

**Comment by w-bruns** [comment]
Well, it seems that the operator = for renf_class does not copy the generator name.

**Comment by videlec** [comment]
Thanks for the report. Fixed in #125.
